src/MediaManagerBundle/DependencyInjection/Compiler/ReplaceUrlGeneratorPass.php

```diff
@@ -1,59 +0,0 @@
-<?php
-
-/*
- * This file is part of the Chameleon System (https://www.chameleonsystem.com).
- *
- * (c) ESONO AG (https://www.esono.de)
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace ChameleonSystem\MediaManagerBundle\DependencyInjection\Compiler;
-
-use ChameleonSystem\MediaManagerBundle\DependencyInjection\Configuration;
-use LogicException;
-use Symfony\Component\Config\Definition\Processor;
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
-use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
-use Symfony\Component\DependencyInjection\Reference;
-
-class ReplaceUrlGeneratorPass implements CompilerPassInterface
```

Comment:
What was this doing?

(It's not deprecated?)

Reply:
The pass replaced a class in a service as soon as the bundle was installed. This is no longer required, as the class from this bundle is now the default (defined as a service in services.xml, configured in the bundle extension class).

As written above, there were some classes for which the @deprecated annotation was forgotten - we need to decide if we remove those (I'd say it's OK).